Makefile

```diff
@@ -89,6 +90,10 @@ gen-latest-deploy-yaml:
 	@echo "Generating latest deployment yaml file"
 	hack/gen-deployment-yaml.sh
 
+gen-latest-prometheus-rules-yamls:
```

Comment:
add lint checks to ensure we generate valid rules.

Reply:
ack